crates/core/src/sql/ast.rs

```diff
@@ -283,7 +281,7 @@ pub enum SqlAst {
     },
     SetVar {
         name: String,
-        value: AlgebraicValue,
+        literal: String,
     },
```

Comment:
Same here. If is a reason to  switch to stringy values....

Reply:
Yes and no. For the AST, a string makes sense because we haven't started type checking or variable binding. But once we have a CrudExpr, then yes I agree, types should already be resolved and we shouldn't need to pass around unparsed strings. I've saved that part for future work when we update type checking in general.